<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongContsEditor</class>
 <widget class="QWidget" name="SongContsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="labelCoverArt">
     <property name="text">
      <string>cover</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPlay">
     <property name="text">
      <string>play</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTime">
     <property name="text">
      <string>time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelArtist">
     <property name="text">
      <string>artist</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAlbum">
     <property name="text">
      <string>album</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelGenre">
     <property name="text">
      <string>genre</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelMore">
     <property name="text">
      <string>more</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
